app/controllers/transcriptions_controller.rb

```diff
@@ -17,15 +17,16 @@ def show
     @transcription = Transcription.find(params[:id])
     authorize @transcription
 
+    if TranscriptionPolicy.new(current_user, @transcription).has_update_rights? &&
+       !@transcription.locked?
```

Comment:
Run the least expensive operation in boolean logic first (https://en.wikipedia.org/wiki/Short-circuit_evaluation)
```suggestion
    if !@transcription.locked? && TranscriptionPolicy.new(current_user, @transcription).has_update_rights?
```